graphql/schemas/Ecosystem/apps.graphql

```diff
@@ -155,4 +186,12 @@ extend type Mutation @guardByAppKey {
         @field(
             resolver: "App\\GraphQL\\Ecosystem\\Mutations\\Apps\\AppUserManagementMutation@updateEmail"
         )
+    saveS3Settings(id: String!, input: s3FilesystemConfigInput): Mixed!
```

Comment:
@arfenis add them to postman

Reply:
@kaioken what about first approve the pr before add them to postman :)